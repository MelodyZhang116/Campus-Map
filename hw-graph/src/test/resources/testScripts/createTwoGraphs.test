# test that a graph that is created can have no influence on the graghs that exist.

# Create a graph
CreateGraph graph1

# Print the nodes in the graph
ListNodes graph1

# Create another graph
CreateGraph graph2

# Print the nodes in the second graph
ListNodes graph2

# Print the nodes in the first graph to make sure it is not affected
ListNodes graph1