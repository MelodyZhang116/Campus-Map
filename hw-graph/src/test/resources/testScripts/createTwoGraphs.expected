# test that a graph that is created can have no influence on the graghs that exist.

# Create a graph
created graph graph1

# Print the nodes in the graph
graph1 contains:

# Create another graph
created graph graph2

# Print the nodes in the second graph
graph2 contains:

# Print the nodes in the first graph to make sure it is not affected
graph1 contains: